Symfony/CS/Finder/DefaultFinder.php

```diff
@@ -50,6 +50,8 @@ public function setDir($dir)
     /**
      * Gets the directories that needs to be scanned for files to validate.
      *
+     * @param $dir
```

Comment:
Well, adding the missing ``@param`` is a good idea. but it should be added in a valid way